examples/with-mobx/components/Clock.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { inject, observer } from 'mobx-react'
+
+const Clock = inject('store')(observer(({store}) => {
```

Comment:
We can, since The Clock element is used in a page component. However in a real app case, the component could be nested in multiple layers.